pkg/reconciler/instances/istio/action.go

```diff
@@ -3,7 +3,6 @@ package istio
 import (
```

Comment:
In general I don't like the fact that all these actions are mixed in the same file. The tests are huge then because they are all defined in the same testing file too.